services/core/BACnetProxy/bacnet_proxy/agent.py

```diff
@@ -357,14 +379,24 @@ def indication(self, apdu):
         # forward it along
         BIPSimpleApplication.indication(self, apdu)
 
+    # Handler for ConfirmedCOVNotificationRequests, forwards the notification to the appropriate driver agent
```

Comment:
Make this a python doc entry rather than a comment.  Ideally point to the documentation of the external bacpypes if possible.